launch-script/src/proposals/create_treasury_pool.rs

```diff
@@ -0,0 +1,41 @@
+// =========================================================================
+// Create collateral pool accounts
+// =========================================================================
+
+use crate::prelude::*;
+
+pub fn create_collateral_pool_accounts(_wallet: &Wallet, _client: &Client,
+        transaction_inserter: &mut ProposalTransactionInserter,
+        cfg: &Configuration,
+        evm_loader_pubkey: Pubkey,
+        collateral_pool_base_pubkey: Pubkey,
+) -> Result<(), ScriptError> {
+
+    let mut seed: String = String::with_capacity(25);
+    let minimum_balance_for_rent_exemption = cfg.client.get_minimum_balance_for_rent_exemption(0).unwrap();
+
+    for index in 0u32..10 {
+        seed.push_str("collateral_seed_");
+        seed.push_str(index.to_string().as_str());
```

Comment:
Maybe better `let seed = format!("collateral_seed_{}", index)`?

Reply:
The suggested variant is simpler. Fixed.